# Fix logcat errors
# android.system.suspend-service: Error opening kernel wakelock stats for: wakeup26 (../../devices/platform/soc/880000.i2c/i2c-5/5-0028/wakeup/wakeup26): Permission denied
# android.system.suspend-service: Error opening kernel wakelock stats for: wakeup0 (../../devices/platform/soc/c440000.qcom,spmi/spmi-0/spmi0-00/c440000.qcom,spmi:qcom,pm8150@0:qcom,power-on@800/wakeup/wakeup0): Permission denied
# android.system.suspend-service: Error opening kernel wakelock stats for: wakeup24 (../../devices/platform/soc/c440000.qcom,spmi/spmi-0/spmi0-00/c440000.qcom,spmi:qcom,pm8150@0:qcom,pm8150_rtc/wakeup/wakeup24): Permission denied
# android.system.suspend-service: Error opening kernel wakelock stats for: wakeup22 (../../devices/platform/soc/894000.qcom,qup_uart/wakeup/wakeup22): Permission denied
# android.system.suspend-service: Error opening kernel wakelock stats for: wakeup19 (../../devices/platform/soc/88e0000.qcom,msm-eud/wakeup/wakeup19): Permission denied
# android.system.suspend-service: Error opening kernel wakelock stats for: wakeup25 (../../devices/platform/soc/c440000.qcom,spmi/spmi-0/spmi0-00/c440000.qcom,spmi:qcom,pm8150@0:qcom,pm8150_rtc/rtc/rtc0/wakeup25): Permission denied
# android.system.suspend-service: Error opening kernel wakelock stats for: wakeup43 (../../devices/platform/soc/aae0000.qcom,venus/wakeup/wakeup43): Permission denied
diff --git a/sepolicy/qti/vendor/genfs_contexts b/sepolicy/qti/vendor/genfs_contexts
--- a/sepolicy/qti/vendor/genfs_contexts
+++ b/sepolicy/qti/vendor/genfs_contexts
@@ -157,3 +157,12 @@ genfscon sysfs /devices/platform/soc/soc:qcom,pmic_glink/soc:qcom,pmic_glink:qco
 genfscon sysfs /devices/platform/soc/soc:qcom,pmic_glink/soc:qcom,pmic_glink:qcom,battery_charger/soc:qcom,pmic_glink:qcom,battery_charger:oplus,pm8350_charger:oplus,mms_wired/oplus_mms/wired/wakeup u:object_r:sysfs_wakeup:s0
 genfscon sysfs /devices/platform/soc/soc:qcom,pmic_glink/soc:qcom,pmic_glink:qcom,battery_charger/usb/wakeup u:object_r:sysfs_wakeup:s0
 genfscon sysfs /devices/platform/vendor/vendor:motor_pl/wakeup u:object_r:sysfs_wakeup:s0
+
+# Additional Wakeup
+genfscon sysfs /devices/platform/soc/880000.i2c/i2c-5/5-0028/wakeup/wakeup26 u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/platform/soc/c440000.qcom,spmi/spmi-0/spmi0-00/c440000.qcom,spmi:qcom,pm8150@0:qcom,power-on@800/wakeup/wakeup0 u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/platform/soc/c440000.qcom,spmi/spmi-0/spmi0-00/c440000.qcom,spmi:qcom,pm8150@0:qcom,pm8150_rtc/wakeup/wakeup24 u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/platform/soc/894000.qcom,qup_uart/wakeup/wakeup22 u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/platform/soc/88e0000.qcom,msm-eud/wakeup/wakeup19 u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/platform/soc/c440000.qcom,spmi/spmi-0/spmi0-00/c440000.qcom,spmi:qcom,pm8150@0:qcom,pm8150_rtc/rtc/rtc0/wakeup25 u:object_r:sysfs_wakeup:s0
+genfscon sysfs /devices/platform/soc/aae0000.qcom,venus/wakeup/wakeup43 u:object_r:sysfs_wakeup:s0
